app/src/main/java/com/xa/xpensauditor/AddTransactionActivity.java

```diff
@@ -271,7 +271,7 @@
 
 
                 RefTran1.addChildEventListener(new com.firebase.client.ChildEventListener() {
-                    String amount, cat, shname, shDay, shMonth, shYear;
+                    String amount, cat, shname, shDay, shMonth, shYear,x ;
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.coding.MultipleVariableDeclarationsCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Each variable declaration must be in its own statement.